index.html

```diff
@@ -384,43 +400,56 @@ <h1>Skills</h1>
 </div>
 </div>
 
-<div class="form">
-  <form>
+
+
+  <div class="form">
+  <form action="https://formspree.io/f/xqkoqrpz" method="POST" id="form" class="contactform">
    <div class="fullname">
-    <input type="text" name="" id="fullname"placeholder="Full Name"><br><br>
+    <input type="text" name="name" id="fullname"placeholder="Full Name" size="40" maxlength="30" required><br><br>
   </div>
   <div class="email">
-    <input type="email" name="" id="email"placeholder="Email address"><br><br>
+    <input type="email" name="email" id="email"placeholder="Email address" size="40" maxlength="30" required><br><br>
   </div>
   <br><br>
   <div class="textareamanage">
-    <textarea name="" id="textarea" cols="41" rows="10" maxlength="500" placeholder="Enter text Here"></textarea>
+    <textarea name="description" id="textarea" cols="41" rows="3" maxlength="500" placeholder="Enter text Here"required ></textarea>
   </div>
   </form>
   <div class="insideautotouchbutton">
   <div class="getintouch">
-    <button class="getin">Get in Touch</button>
+    <button type ="submit" class="getin">Get in Touch</button>
```

Comment:
- [ ] I have noticed that the form can not be submitted as intended. This is because the submit button has been placed in the wrong position. Kindly verify the button position so that when clicked on the form is submitted and data  collected by formsphere ☺️ .
- [ ] You have done incredibly well to ensure the design that matches the Figma. However, you need to ensure that you stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=XJ6goFyUx0B0uoSu-0) as much as possible. You may need to ensure that when the button is pressed the correct color is shown as seen in the Figma design ☺️ .

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/221547589-2e7031a6-9d5f-4e16-859f-07a56848102a.png)